scipy/stats/_discrete_distns.py

```diff
@@ -28,9 +28,11 @@ class binom_gen(rv_discrete):
 
        f(k) = \binom{n}{k} p^k (1-p)^{n-k}
 
-    for ``k`` in ``{0, 1,..., n}``.
+    for ``k`` in ``{0, 1,..., n}``, :math:'0 \leq p \leq 1'
```

Comment:
Here and in the other changes, the delimiters must be backticks, not single quotes:
```suggestion
    for ``k`` in ``{0, 1,..., n}``, :math:`0 \leq p \leq 1`
```

Reply:
Ah, I see, thank you!